### PR TITLE
ClusterLoader- Fixing measurements for kubemark

### PR DESCRIPTION
**Pull request by krzysied**

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/simple/resource_usage.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func (e *resourceUsageMetricMeasurement) Execute(config *measurement.Measurement
 		}
 
 		glog.Infof("%s: starting resource usage collecting...", e)
-		e.gatherer, err = gatherers.NewResourceUsageGatherer(config.ClientSet, provider, host, gatherers.ResourceGathererOptions{
+		e.gatherer, err = gatherers.NewResourceUsageGatherer(config.ClientSet, host, provider, gatherers.ResourceGathererOptions{
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/resource_usage.go]: This is a bit misleading (if you look into below fine, where the order is "provider, host - line 149 scheduler_latency.go).
Can we agree on one order of those and use everywhere?

Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/resource_usage.go]: Changed to "host, provider" in every file.

**Pull request merged**